app/models/order_item.rb

```diff
@@ -0,0 +1,13 @@
+class OrderItem < ActiveRecord::Base
+  belongs_to :order
+  belongs_to :ticket_type
+
+  validates :quantity, presence: true, numericality: { greater_than: 0 }
```

Comment:
Always a good rule of thumb to have validations at the model levels, since that will also work for potential hackers, mobile devices, etc...